pkg/asset/manifests/cluster_k8s_io.go

```diff
@@ -0,0 +1,62 @@
+package manifests
```

Comment:
already have this file here `pkg/assets/machines/...`

Reply:
import loop. had to move it.

Reply:
split commits so that's visible. and include description of how there is an import loop. I want to keep it in the same package as much as possible.

Reply:
hm, guess enough of the file changed that it didn't trigger the git rename heuristic. neat.